src/main/g8/build.sbt

```diff
@@ -3,8 +3,7 @@ val scala212Version = "2.12.2"
 
 lazy val root = (project in file(".")).
   settings(
-    name := "dotty-cross-template",
-    description := "Template sbt project that is cross compiled using Dotty and Scala 2",
```

Comment:
while technically this is useless, it would be nice to provide some breadcrumbs for people who don't know sbt or g8 well to see what happens here.